checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java

```diff
@@ -19,8 +19,8 @@
  * <p>You can use the following {@link SuppressWarnings} prefixes with this checker:
  *
  * <ul>
- *   <li>{@code @SuppressWarnings("nullness")} suppresses warnings for both nullness and
- *       initialization annotations
+ *   <li>{@code @SuppressWarnings("nullness")} suppresses warnings for nullness, initialization and
```

Comment:
Ah, so I had mixed up the suppression logic. Do you think it is nicer to suppress all three or should we stick with the current logic?